lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -171,11 +171,15 @@ createClangArgs(const ASTContext &ctx, clang::driver::Driver &clangDriver) {
   return clangDriverArgs;
 }
 
+static bool shouldInjectGlibcModulemap(const llvm::Triple &triple)
+{
+  return triple.isOSGlibc() || triple.isOSOpenBSD() || triple.isOSFreeBSD();
```

Comment:
[This will break the native Android build](https://github.com/llvm/llvm-project/blob/b2f31cac28c8a03ceb908b544f5790f4f9f2d9ab/llvm/include/llvm/ADT/Triple.h#L647), please add Android here and maybe a comment explaining why we're adding these non-glibc platforms.

Reply:
Done; can you pick and retest?